controller_manager:
  ros__parameters:
    update_rate: 100  # Hz

    joint_state_broadcaster:
      type: joint_state_broadcaster/JointStateBroadcaster

    simple_velocity_controller:
      type: velocity_controllers/JointGroupVelocityController

    swerve_steering_controller:
      type: position_controllers/JointGroupPositionController

simple_velocity_controller:
  ros__parameters:
    joints:
      - fr_wheel_joint
      - fl_wheel_joint
      - rl_wheel_joint
      - rr_wheel_joint

swerve_steering_controller:
  ros__parameters:
    joints:
      - steer_fl_joint
      - steer_fr_joint
      - steer_rl_joint
      - steer_rr_joint
